specification/compute/resource-manager/Microsoft.Compute/stable/2020-06-01/compute.json

```diff
@@ -8665,6 +8716,10 @@
           "$ref": "#/definitions/SubResource",
           "description": "Specifies information about the dedicated host that the virtual machine resides in. <br><br>Minimum api-version: 2018-10-01."
         },
+        "hostGroup": {
+          "$ref": "#/definitions/SubResource",
+          "description": "Specifies information about the dedicated host group that the virtual machine resides in. <br><br>Minimum api-version: 2020-06-01."
```

Comment:
Also, mention a NOTE in bold letters that user cannot specify both the host and hostGroup properties.